### PR TITLE
[MM-40022] - Expose telemetry service in the server

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR exposes the telemetry service so it can be used outside of the telemetry service package

#### Ticket Link
[MM-40022](https://mattermost.atlassian.net/browse/MM-40022)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by AGMETEOR** [comment]
Hello @marianunez mentioning you here to let you know that we need to do this if we're to have telemetry in the invite email job here https://github.com/mattermost/mattermost-server/pull/18894

File: app/app.go [hunk 1]
```diff
@@ -84,6 +85,9 @@ func (s *Server) getFirstServerRunTimestamp() (int64, *model.AppError) {
 func (a *App) Srv() *Server {
 	return a.ch.srv
 }
+func (a *App) GetTelemetryService() *telemetry.TelemetryService {
+	return a.ch.srv.telemetryService
+}
```
Comment (agnivade) [inline app/app.go:90]: It's better to have a getter on the `*Server` struct and return that instead.

**Review by agnivade — COMMENTED** [review]
Looks good. Just a small comment.

**Review by agnivade — APPROVED** [review]

**Review by jespino — APPROVED** [review]
LGTM

**Comment by AGMETEOR** [comment]
Hi @amyblais this is part of the Invite email A/B test PR. I think we should cherry pick to cloud as well/

**Comment by AGMETEOR** [comment]
Looks like this is already on cloud branch :)

**Pull request merged**